### ISSUE TITLE
Media shortcuts can't control Bluetooth playback

### ISSUE THREAD

**Issue by GranPC** [elementary/os#210]
Not sure where to file this.

I tried using the media shortcuts to control the music that's playing on my phone, while nothing is playing locally, expecting it to work just like the sound indicator, but instead it just shows up a (fairly ugly) notification letting me know nothing is playing back right now.

![Screenshot from 2019-07-02 12-33-27](https://user-images.githubusercontent.com/1385480/60506254-c8087a80-9cc5-11e9-8ee4-713adde4699a.png)

**Comment by peteruithoven** [comment]
I don't think that will be possible. If I understand it correctly we're even working on removing phones from that list: https://github.com/elementary/wingpanel-indicator-sound/pull/104

**Comment by GranPC** [comment]
We're not removing phones from that list, we're just hiding the microphone indicators that shouldn't be appearing in the first place. See the issue at elementary/wingpanel-indicator-sound#103.

I do believe this is possible; I just have no idea which package is responsible for handling media keys. If you give me some pointers I might be able to do this myself. :)

**Comment by peteruithoven** [comment]
Alright, I believe it's a plugin for the gnome-settings-daemon:
https://gitlab.gnome.org/GNOME/gnome-settings-daemon/tree/master/plugins/media-keys

**Comment by GranPC** [comment]
Great, thank you so much. I'll look into it!

**Comment by danrabbit** [comment]
I wonder if it makes sense for the sound indicator to provide a MPRIS interface for Bluetooth devices. I think that would give us media keys support and expose the device as an audio player to anything else listening to MPRIS like maybe a lyrics app or the like

**Comment by GranPC** [comment]
That seems like a very elegant solution, I hadn't thought of that.
